apps/server/src/modules/task-card/uc/task-card.uc.spec.ts

```diff
@@ -467,7 +467,20 @@ describe('TaskCardUc', () => {
 				await uc.update(user.id, taskCard.id, failingTaskCardUpdateParams);
 			}).rejects.toThrow(ValidationError);
 		});
-		it('should throw if course end date is missing and dueDate after schoolYearEnd ', async () => {
+		it('should not throw if courseEndDate is before dueDate but only regarding the time not the date ', async () => {
+			course = courseFactory.buildWithId({ untilDate: new Date(tomorrow.setHours(23, 58)) });
+			user = userFactory.buildWithId();
+			courseRepo.findById.mockResolvedValue(course);
+			taskCardUpdateParams = {
+				title,
+				visibleAtDate: new Date(Date.now()),
+				dueDate: new Date(tomorrow.setHours(23, 59)),
+				courseId: course.id,
+			};
+			const { card } = await uc.update(user.id, taskCard.id, taskCardUpdateParams);
+			expect(card.dueDate).toEqual(taskCardUpdateParams.dueDate);
+		});
+		it('should throw if courseEndDate is missing and dueDate after schoolYearEnd ', async () => {
```

Comment:
@virgilchiriac can I write the test like this or is this against any convention?